src/Compilers/Core/Portable/Symbols/IFieldSymbol.cs

```diff
@@ -39,6 +39,13 @@ public interface IFieldSymbol : ISymbol
         /// </summary>
         bool IsVolatile { get; }
 
+        /// <summary>
+        /// Returns true if this field was declared as "fixed".
+        /// Note that for a fixed-size buffer declaration, this.Type will be a pointer type, of which
+        /// the pointed-to type will be the declared element type of the fixed-size buffer.
+        /// </summary>
+        bool IsFixed { get; }
```

Comment:
>bool IsFixed { get; } [](start = 8, length = 21)

I am assuming this property is implicitly implemented by  C# FieldSymbol.  Please add targeted tests for this new API for C#.

Reply:
👍 Tests now added